### DISCUSSION TITLE
İskambil destesi sihirbazlık sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#123] [Zeka Sorusu]
Filiz in 52 kartlık standard bir iskambil destesi var. Bu kartlarla asistanı ve xenix yardımıyla seyircilerine küçük bir numara yapacak.
Seyircilerden birisi iyice karışmış deste içinden rastgele beş kartı asistanına verecek. Asistan da bu beş karttan birini seçerek cebine koyacak ve diğer dört kartı istediği sırada tek tek sihirbaza gösterecek. Sihirbaz da asistanın cebindeki son kartın ne olduğunu şıp diye söyleyiverecek. Asistan ile sihirbaz arasında kartları göstereceği sıra dışında önceden belirlenmiş bir şifre (mimikler, el hareketleri, vb) olmamalı. Bu numarayı yapmak için nasıl bir yöntem izlemeliler?

**Comment by MnG-Official** [comment]
Hocam bence olay sıralama ve kartları tutuş açısıyla alakalı. Eskiden ben de arkadaşımla buna benzer bir numara yapmıştım ama tam olarak böyle bir şey değildi fakat burada da aynı metodu kullanabiliriz gibime geliyor. Biz o zaman şu şekilde yapmıştık:
*1. kart rengi belirliyor.
*2. kart, 5. kartın tek mi yoksa çift mi olduğunu ve aynı zaman da 3. kart sayesinde kartın üzerinde yazan sayının ne olduğunu belirliyor.
*3. kart dediğim gibi yardımcı karttır. 2. karta destekte bulunur.
*4. kart ise şekli belirlemede yardımcı olabilir bu sayede iyice çalışıldıktan sonra gayet güzel bir gösteri olacağını düşünüyorum. Ama bu 3. kartın destek vermesi konusuna gerçekten dikkat edilmeli, tutuş biçimi, diğer karta yakınlığı etc. gerçekten dikkatle gözlemlenmeli ve çok çalışılmalı.
Tabii ki bir de ortada joker kartlar var bu da işi bozan taraf yani ben olsam joker kartı cebime koymak yerine göstermeyi tercih ederim çünkü joker kartları tahmin etmek zordur.
